server_admin/topics/roles-groups/con-role-scope-mappings.adoc

```diff
@@ -8,12 +8,24 @@ On creation of an OIDC access token or SAML assertion, the user role mappings be
 
 _Role Scope Mappings_ limit the roles declared inside an access token.  When a client requests a user authentication, the access token they receive contains only the role mappings that are explicitly specified for the client's scope.  The result is that you limit the permissions of each individual access token instead of giving the client access to all the users permissions.  
 
-By default, each client gets all the role mappings of the user. You can view the role mappings in the *Scope* tab of each client.
+By default, each client gets all the role mappings of the user.
+ifeval::[{project_community}==true]
+You can view the role mappings in the *Scope* tab of each client.
+. Click *Clients* in the menu
+. Click the client to go to the details.
+. Click the *Client scopes* tab.
+. Click *<client-name>-dedicated* scope.
```

Comment:
I think this is supposed to be:
. Click the link under *Assigned client scope*.
 OR the link in the row with
*Dedicated scope and mappers for this client*